backend/slackbot/handler.js

```diff
@@ -1,91 +1,231 @@
 'use strict';
-const hook = 'https://hooks.slack.com/services/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';
+//const hook = 'https://hooks.slack.com/services/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'; // testing slack
+const hook = 'https://hooks.slack.com/services/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'; // group slack
 
 const gatewayURL = process.env.GW_URL;
-
 const axios = require('axios');
+const queryString = require('query-string');
 
 module.exports = {
   handler,
+  handleCommand,
   getApps,
   getStatistics,
-  sendStats
+  sendStats,
+  getSentimentOverTime,
 };
 
-async function handler () {
+async function handler (event) {
+  let body = event.body;
+  let responseMessage = '';
+
+  // Generate report or slash command
+  if (event.httpMethod == 'POST') {
+    try {
+      responseMessage = await handleCommand(body);
+    } catch (error) {
+      return {
+        statusCode: 500,
+        error: `Error handeling post request: ${error}`
+      };
+    }
+  } else {
+    try {
+      responseMessage = await report();
+    } catch (error) {
+      return {
+        statusCode: 500,
+        error: `Error Generating Report: ${error}`
+      };
+    }
+  }
+
+  console.log(responseMessage);
```

Comment:
I was referring to the `console.log`, but we should definitely look at our HTTP responses as well. If we need the message in the response to send back to Slack (such as in response to slash commands), we should leave it, but otherwise a 200 response without any other message is probably fine.